README.md

```diff
@@ -1 +1,49 @@
-# MediaMonks - Frontend Coding Standards
+# frontend-coding-standards
+
+## Recommended Libraries
+
+#### Animation & UI
+ * [classnames](https://www.npmjs.com/package/classnames) - A simple JavaScript utility for conditionally joining classNames together.
+ * [Draggable](https://greensock.com/draggable/) - Green Sock library for dragging elements
```

Comment:
Draggable is part of gsap. So perhaps merge those two?

Reply:
Depends if we go for listing use cases or libraries :D

I think it's more useful to list it like; I have a problem / use case... which library should I use (or can I pick from, depending on certain criteria and conditions)?

Instead of, let's include this library, but what can I do with it? - seems silly :)